.travis.yml

```diff
@@ -25,10 +25,13 @@ matrix:
             - cmake-data
             - xsltproc #only for doc transform to html , can be removed
             - valgrind
+            - wget # for downloading artifacts
       env:
         - MATRIX_EVAL="CC=gcc-8 && CXX=g++-8"
         - PLATFORM="linux"
         - PYTHONPATH="$PYTHONPATH:$HOME/.local/lib/python3.7/site-packages/"
+        - WINDOWS_ARTIFACT_URL="https://ci.appveyor.com/api/buildjobs/by4cpqmul79tp9ag/artifacts/build/scripts/htm_core-6866bd13ece35608313709e82c29b2260ee7be73-windows64.zip" #FIXME make the URLs automated, needs API?
+        - OSX_ARTIFACT_URL="https://3238-118031881-gh.circle-artifacts.com/0/tmp/uploads/htm_core-7b7b53c798b3795088ce772e96c6994068fc1302-darwin64.tar.gz"
```

Comment:
With the PR #632 you can use the contents of file VERSION as the version part of the release distribution file name.

I assume we do not distribute unless there is a new tag.

If the PR was tagged (not the master) then the VERSION file will contain the value from the tag (assuming it was just built).   When the PR is merged to the master, the master will get the tag.

Reply:
>  can use the contents of file VERSION as the version part of the release distribution file name

that's great! 

> I assume we do not distribute unless there is a new tag.

I thought we could, not to deploy releases, but just keep the artifacts uploaded. That would still work (?) 

> If the PR was tagged (not the master) then the VERSION file will contain the value from the tag (assuming it was just built). When the PR is merged to the master, the master will get the tag.

this will be a problem. Travis GH releases need tag on master. It's also quite nice, as master is always in "runable, passing" state. So any time one issues a tag on master, release builds start. 
So could the code update version on master tag?